packages/oui-calendar/src/calendar.component.js

```diff
@@ -12,6 +12,10 @@ export default {
         format: "@?",
         altFormat: "@?",
 
+        appendTo: "<?",
+        inline: "<?",
+        "static": "<?",
```

Comment:
Careful with this one. It create another input in a `div.flatpickr-wrapper` that is not considered by Angular.
My workaround was to use `appendTo` instead: https://github.com/ovh-ux/ovh-ui-angular/pull/146/files

I think we should just support `appentTo` and `inline`, and avoid `static`. To be discussed :)